src/pages/host/Metadata.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { ApolloError } from "apollo-client";
+import { StyledLink } from "components/styles";
+import { wordBreakCss, gray } from "components/Typography";
+import { HostMetaDataCard } from "components/HostMetaDataCard";
+import { HostQuery } from "gql/generated/types";
+import { getUiUrl } from "utils/getEnvironmentVariables";
+import styled from "@emotion/styled/macro";
+import { getDateCopy } from "utils/string";
+
+export const Metadata: React.FC<{
+  loading: boolean;
+  data: HostQuery;
+  error: ApolloError;
+}> = ({ loading, data, error }) => {
+  const host = data ? data.host : null;
```

Comment:
`const host = data?.host ?? null`

Reply:
you can also do `const host = data?.host`